types/chrome-har-capturer/index.d.ts

```diff
@@ -136,7 +136,7 @@ export interface RunOptions<T> {
      * @param urls input URL array
      * @returns Promise executed before each page load
      */
-    preHook?: (url: string, client: any, index: number, urls: readonly string[]) => Promise<void>;
+    preHook?: (url: string, client: any, index: number, urls: readonly string[]) => Promise<void> | undefined;;
```

Comment:
I think you meant:

```suggestion
    preHook?: ((url: string, client: any, index: number, urls: readonly string[]) => Promise<void>) | undefined;
```

Reply:
Yes, got duplicated while pasting within GitHub